File: Homework_1/Java/7.java
import java.util.Scanner;
class Swap{
	public static void main(String[] args)
	{
		Scanner a = new Scanner(System.in);
        int x = a.nextInt();
		Scanner b = new Scanner(System.in);
		int y = b.nextInt();
        int temp = x;
        x = y;
        y = temp;
        System.out.printf("Swapped x and y: %d, %d",x ,y);
        a.close();
        b.close();
    }
}

File: Homework_1/CPP/7.cpp
#include <iostream>

int main()
{
    int a,b;
    std::cout << "a:";
    std::cin >> a;
    std::cout << "b:";
    std::cin >> b;
    int temp;
    
    temp = a;
    a = b;
    b = temp;

    std::cout << "Swapped a and b: " << a << ", " << b << std::endl;
    return 0;
}
